.github/workflows/deploy_to_aws.yml

```diff
@@ -248,11 +253,11 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DEFAULT_REGION: "eu-west-1"
 
-  restore_database:
+  stop_logstash:
     needs: create_kafka_topic
-    runs-on: self-hosted
+    runs-on: ubuntu-latest
     environment: ${{ inputs.deploy-env || (github.ref_name == 'main' && 'Development') || (startsWith(github.ref_name, 'releases/') && 'Pre-prod') || (startsWith(github.ref_name, 'production-') && 'Production') || (startsWith(github.ref_name, 'sandbox-') && 'Staging') || 'None' }}
-    if:  ${{ inputs.deploy-plan-only == false }}
+    if: ${{ inputs.deploy-plan-only == false }}
```

Comment:
Please, add a check to ensure it only starts if `restore-db` or `clear-opensearch` are `true`. Without this, we will stop Logstash even when we don't need to clear OpenSearch, which is unnecessary.

Reply:
We discussed that at the post-deployment stage there may be migrations affecting the data schema and therefore Logstash should not be running at this time. Will there be a situation where the database does not need to be restored, but migrations will still occur?

Reply:
There are two cases to consider:

1. If a developer knows that OpenSearch needs to be reindexed due to changes from DB migrations or OpenSearch index mapping modifications, they should set the "clear opensearch" flag via the `Deploy to AWS` or `Release [Deploy]` workflows. Only then should this job be activated.
2. If the "restore db" flag is set in the `Deploy to AWS` workflow, it means that Logstash and OpenSearch should be cleared.

In all other cases, this job should not be activated.

Reply:
Why can't this check be done on the job level?

Reply:
if we add this check on job level, all followed jobs be skipped every time when triger not be set to true

Reply:
Got it, thank you!